pkg/uevent/uevent.go

```diff
@@ -35,7 +34,7 @@ type ReaderCloser struct {
 func NewReaderCloser() (io.ReadCloser, error) {
 	nl := unix.SockaddrNetlink{
 		Family: unix.AF_NETLINK,
-		Pid:    uint32(os.Getpid()),
+		Pid:    uint32(0),
```

Comment:
I appreciate you've documented this in the commit, but I'd also be tempted to put a comment above this line as it looks "odd" ;)